roles/vector/tasks/install.yml

```diff
@@ -1,46 +1,31 @@
 ---
-- block:
-    # Workaround for latest version being named against a version number
-    - name: Get latest version
-      uri:
-        url: https://s3.amazonaws.com/packages.timber.io/?prefix=vector/latest&max-keys=1
-        return_content: true
-      register: bucket_content
-      when: vector_version == "latest"
 
-    - name: Set latest version
-      set_fact:
-        vector_version: "{{ bucket_content.content | regex_replace('.*\\n.*<Key>[^-]+-(?P<version>[\\d\\.]+)-.*','\\g<version>') }}"
-      when: vector_version == "latest"
-
-    - name: Install Vector (Debian)
-      apt:
-        deb: "https://packages.timber.io/vector/{{ version }}/vector-{{ version }}-{{ arch }}.deb"
+- name: Install vector from source
```

Comment:
with "install from source" i'd expect a compilation step here. I guess "install vector directly from package" would be a better description for what is happening here

Reply:
changed :)